driver/ipam_driver.go

```diff
@@ -2,28 +2,28 @@ package driver
 
 import (
 	"fmt"
-	"log"
 	"net"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
Import this as "log" e.g. 
`log "github.com/Sirupsend/logrus"`

Then you don't need to make so many changes elsewhere